src/settings.js

```diff
@@ -5,14 +5,12 @@ var path = require('path-extra');
 
 var settings = {};
 
-var getSettingsDir = function(homedir){
-  homedir = homedir || path.homedir();
-  return path.join(homedir, '.matterfront');
+var getStatePath = function(userDataPath){
+  return path.join(userDataPath, 'state.json');
 };
 
-var getStatePath = function(homedir){
-  var settingsDir = getSettingsDir(homedir);
-  return path.join(settingsDir, 'state.json');
+var getConfigPath = function(appPath){
+  return path.join(appPath, 'config.json');
```

Comment:
I'll have to rename this to `state.json`, so as not to be confusing with the read-only `config.json` that lies in `userDataPath`.